src/gsad_gmp.c

```diff
@@ -3112,6 +3115,29 @@ create_credential_gmp (gvm_connection_t *connection, credentials_t *credentials,
                   credential_login ? credential_login : "",
                   password ? password : "", allow_insecure);
         }
+      else if (str_equal (type, "krb5"))
+        {
+          CHECK_VARIABLE_INVALID (credential_login, "Create Credential");
+          CHECK_VARIABLE_INVALID (password, "Create Credential");
+          CHECK_VARIABLE_INVALID (kdc, "Create Credential");
+          CHECK_VARIABLE_INVALID (realm, "Create Credential");
+
+          ret = gmpf (
+            connection, credentials, &response, &entity, response_data,
+            "<create_credential>"
+            "<name>%s</name>"
+            "<comment>%s</comment>"
+            "<type>%s</type>"
+            "<login>%s</login>"
+            "<password>%s</password>"
+            "<kdc>%s</kdc>"
+            "<realm>%s</realm>"
+            "<allow_insecure>%s</allow_insecure>"
+            "</create_credential>",
+            name, comment ? comment : "", type,
+            credential_login ? credential_login : "", password ? password : "",
+            kdc ? kdc : "", realm ? realm : "", allow_insecure);
```

Comment:
Do you think the checks here are still required ? Given that `CHECK_VARIABLE_INVALID` checks for NULL.

Reply:
For the sake of consistency I would like to leave it as it is. These checks also occur at all the other similar places.